templates/sag-cc-all-layer-defs/template.yaml

```diff
@@ -0,0 +1,265 @@
+                        "@alias": sag-tdb-server 
+
+                CCE-LAYER-TYPES-APAMA-CORRELATOR:
+                  "@name": APAMA-CORRELATOR
+                  Description: Create Apama Correlator
+                  Type: RUNTIME
+                  Scalable: 'false'
+                  Templates:
+                      Template:
+                        "@alias": sag-apama-correlator 
+
+                CCE-LAYER-TYPES-EXX-BROKER:
+                  "@name": EXX-BROKER
+                  Description: Create EntireX Broker
+                  Type: RUNTIME
+                  Scalable: 'false'
+                  Templates:
+                      Template:
+                        "@alias": sag-exx-broker 
+
+                CCE-LAYER-TYPES-EXX-C-RPC-SERVER:
+                  "@name": EXX-C-RPC-SERVER
+                  Description: Create EntireX C RPC Server
+                  Type: RUNTIME
+                  Scalable: 'false'
+                  Templates:
+                      Template:
+                        "@alias": sag-exx-broker 
+
+                CCE-LAYER-TYPES-APIGATEWAY-SERVER:
+                  "@name": APIGATEWAY-SERVER
+                  Description: Create API Gateway server
+                  Type: RUNTIME
+                  Scalable: 'true'
+                  Templates:
+                      Template:
+                        "@alias": sag-apigateway-server 
+
+                CCE-LAYER-TYPES-APIGATEWAY-CLUSTER:
+                  "@name": APIGATEWAY-CLUSTER
+                  Description: Create API Gateway cluster
+                  Type: RUNTIME
+                  Scalable: 'true'
+                  Templates:
+                      Template:
+                        "@alias": sag-apigateway-cluster  
+                                                
+                CCE-LAYER-TYPES-INTERNALDATASTORE:
+                  "@name": INTERNALDATASTORE
+                  Description: Create Internal Data Store server or cluster
+                  Type: RUNTIME
+                  Scalable: 'true'
+                  Templates:
+                      Template:
+                        "@alias": sag-internaldatastore  
+                        
+                CCE-LAYER-TYPES-DB-ORACLE-103:
+                  '@name': DB-ORACLE-103
+                  Description: ORACLE database and sample product schemas for 10.3 and lower
+                  Type: DATABASE
+                  Scalable: 'false'
+                  Templates:
+                    Template:
+                      '@alias': sag-db-oracle-103-and-lower   
+                        
+                CCE-LAYER-TYPES-DB-MYSQL-103:
+                  '@name': DB-MYSQL-103
+                  Description: MYSQL server database and sample product schemas for 10.3
+                  Type: DATABASE
+                  Scalable: 'false'
+                  Templates:
+                    Template:
+                      '@alias': sag-db-mysql-103
```

Comment:
is this for 10.3 only or again for lower, too?

Reply:
The previous template was documented as 103+, I think the support started at that time.
But we can check again.